calculator_test.go

```diff
@@ -225,24 +227,28 @@ func TestSqrt(t *testing.T) {
 	}{
 		{
 			name:  "Calculate square root of a positive number",
-			want:  7,
 			input: 49,
 		},
+		{
+			name:  "Calculate square root of a long positive number",
+			input: 94339,
+		},
 		{
 			name:        "Calculate square root of a negative number",
 			errExpected: true,
-			want:        0,
 			input:       -49,
 		},
 	}
 	for _, tC := range testCases {
 		t.Run(tC.name, func(t *testing.T) {
 			got, err := calculator.Sqrt(tC.input)
-			if err != nil && tC.errExpected == false {
-				t.Fatalf("Cannot calculate square root of %f: %s", tC.input, err)
+			errReceived := err != nil
+			if tC.errExpected != errReceived {
+				t.Fatal(err)
 			}
-			if !tC.errExpected && tC.want != got {
-				t.Errorf("want %f, got %f", tC.want, got)
+			want := math.Round(math.Sqrt(tC.input))
```

Comment:
This is dangerous. We should always _pre-calculate_ the test expectation whenever we can, and we can. The reason is that calculating it on the fly leaves us open to at least two potential problems: (1) `math.Sqrt` or `math.Round` might have bugs, or, much more likely, (2) _we_ might have a bug in the way we do the calculation!